app/models/comment.rb

```diff
@@ -0,0 +1,15 @@
+class Comment < ApplicationRecord
+  validates :text, presence: true
+
+  belongs_to :posts
+  belongs_to :users
```

Comment:
- [ ] Let us use the singular form here because of the comment belongs_to one `user` who is going to comment and `post` not `users or posts`.